Team9ProjectTests/src/com/indragie/comput301as1/test/ReceiptTests.java

```diff
@@ -0,0 +1,17 @@
+package com.indragie.comput301as1.test;
+
+import com.indragie.cmput301as1.ExpenseItemAddActivity;
+
+import android.test.ActivityInstrumentationTestCase2;
+
+public class ReceiptTests extends ActivityInstrumentationTestCase2<ExpenseItemAddActivity> {
+
+	public ReceiptTests(String name) {
```

Comment:
I can access resources but I'd need to copy it into a temporary folder somewhere on external storage so that I can get a Uri from it, and then scale the image. The AndroidTestCase wouldn't let me do a couple things though, I'll update the code when I get the chance. On my phone atm